pattern-library/sass/_normalize.scss

```diff
@@ -355,24 +355,12 @@ input[type="number"]::-webkit-outer-spin-button {
  */
 
 input[type="search"] {
-  -webkit-appearance: textfield; /* 1 */
   -moz-box-sizing: content-box;
   -webkit-box-sizing: content-box; /* 2 */
   box-sizing: content-box;
 }
 
 /**
- * Remove inner padding and search cancel button in Safari and Chrome on OS X.
```

Comment:
It's not consistent between browsers (FF and IE don't do it), so maybe somebody was thinking we'd build our own that worked everywhere and would turn the native one off so they didn't conflict?